zipline/pipeline/factors/factor.py

```diff
@@ -1411,6 +1411,13 @@ def __repr__(self):
             mask=self.mask,
         )
 
+    def short_repr(self):
+        return "Rank({})\nmethod={}\nmask={}".format(
```

Comment:
It feels a little odd to me to have some of the parameters to `Rank` be inside parentheses, but to have method and mask outside. What do you think about something more yaml-ish like this?

```
Rank:
  method: 'ordinal'
  mask: SomeFilter 
```

Reply:
Ok, yeah that makes sense